### PR TITLE
📦 : new command aws-accounts

### LINKED ISSUE #3: AWS Account directory -> Allows adding more than 1 user in AWS and change it according the name
something like 

`$ daily-cli aws-accounts add --key="keyhere" --secret="secret-here" --name="aws-test"`
`$ daily-cli aws-accounts to --name aws-test`

### PR DESCRIPTION
**Pull request by AndresMorelos**
Fix #3 

```
USAGE
  $ daily-cli aws-accounts ACTION

ARGUMENTS
  ACTION  (add|remove|show) action that will do the command

OPTIONS
  -k, --key=key        Access Key of the AWS account
  -n, --name=name      Name of the AWS account
  -s, --secret=secret  Secret Access Key of the AWS account

DESCRIPTION
  ...
  Allow administrating your AWS accounts
```

> **NOTE:** The action `to` could not be implemented because to perform a switching AWS profile a change of the machine's environment variable `AWS_DEFAULT_PROFILE` is required, and NodeJS doesn't have a way to execute the command.

### CODE REVIEW

**Comment by AndresMorelos** [comment]
Hey @clouby you will be able to see the changes after a profile addition or removal using the `show` command or accessing to the `~/.aws/credentials` file.

> **NOTE:** for windows the `.aws` folder is on the user home directory

File: src/commands/aws-accounts.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+AWSAccounts.description = `Describe the command here
+...
+Allow to adminsitrate your AWS accounts
+`;
+
+AWSAccounts.flags = {
+  name: flags.string({
+    char: "n",
+    description: "Name of the AWS account",
```
Comment (FlavioAandres) [inline src/commands/aws-accounts.js:109]: just for comment, you can set the default values here instead of using ` || undefined` in the code

Comment (AndresMorelos) [reply src/commands/aws-accounts.js:109]: @FlavioAandres I made the changes to set a default on the flags object for each flag.

**Comment by AndresMorelos** [comment]
@FlavioAandres `to` action was implemented.

**Review by clouby — APPROVED** [review]
Everything works as should be. LGTM :sweat_smile:

**Comment by AndresMorelos** [comment]
@FlavioAandres @clouby `only those with write access to this repository can merge pull requests`

**Review by FlavioAandres — APPROVED** [review]
LGTM, great job @AndresMorelos thanks for your contributions 🤘🏼🤘🏼

**Pull request merged**